contributing/README.md

```diff
@@ -1,23 +1,28 @@
 # Contributing
 
-Before you start contributing to the project, you should familiarize yourself with the [documentation](vmware.github.io/vmware-aria-operations-integration-sdk/),
-[issue tracker](https://github.com/vmware/vmware-aria-operations-integration-sdk/issues), [project structure](index.md),
-and basic functioning of every component.
+Contributing to the project does not necessarily require familiarity with every component. To find a quick breakdown
+of the smaller parts of this project, consult the [index](index.md), which breaks down the SDK Structure into small
+components and references to help contributors find the necessary references without having to go in-depth into the
+project's overall structure. For a more in-depth breakdown of the project structure, it would be beneficial for you
+to have a look at the [documentation](vmware.github.io/vmware-aria-operations-integration-sdk/),
+[issue tracker](https://github.com/vmware/vmware-aria-operations-integration-sdk/issues),
+[project structure](index.md), and basic functioning of every component.
```

Comment:
we shouldn't suggest that someone needs to look at every component to make a contribution.

Reply:
Yup, see my suggestion here: https://github.com/vmware/vmware-aria-operations-integration-sdk/pull/221#discussion_r1256325624